index.html

```diff
@@ -109,6 +121,7 @@
 					}
 				});
 				requestFlows.send();
+				window.localStorage.setItem('FDToken', token);
```

Comment:
Nit, Could the strings have consistent quotation please.  I believe at Resin the preference is for `'` (single quote) where possible.

Reply:
Okay. FYI, there's also inconsistency in the rest of the code, that was there before me, but this PR will be consistent.

Reply:
Eugh, thanks for noticing that.  I will create an issue.